### DISCUSSION TITLE
Tabs position in Preferences dialog

### DISCUSSION THREAD

**Discussion by Toggoren** [zim-desktop-wiki/zim-desktop-wiki#2499] [Polls]
Motivation: 
Today is the era of widescreen monitors.  
Vertical space is much more valuable than horizontal space.  

| PreferencesDialog(current) | PreferencesDialog(proposed change) | PropertiesDialog |
| ------------- | ------------- | ------------- |
| ![image](https://github.com/zim-desktop-wiki/zim-desktop-wiki/assets/94177064/9370f5b4-dfec-4e76-9e26-c34acc643f0c) | ![image](https://github.com/zim-desktop-wiki/zim-desktop-wiki/assets/94177064/16e8c7cc-d1d0-42ae-8230-df89c06a1798) | ![image](https://github.com/zim-desktop-wiki/zim-desktop-wiki/assets/94177064/4d457c86-1113-48a8-b3bb-36d184fb7fac)  |

This is easily done. Just one line.
```diff
diff --git zim/gui/preferencesdialog.py zim/gui/preferencesdialog.py
index ff28859a..517ab649 100644
--- zim/gui/preferencesdialog.py
+++ zim/gui/preferencesdialog.py
@@ -66,2 +66,4 @@ class PreferencesDialog(Dialog):
                gtknotebook = Gtk.Notebook()
+               gtknotebook.set_tab_pos(Gtk.PositionType.LEFT)
+               gtknotebook.set_scrollable(True)
                self.vbox.pack_start(gtknotebook, True, True, 0)

```

**Comment by introt** [comment]
I like how the check boxes and tabs are closer together, reducing required mouse movement.

Do these different positions convey some (possibly culture/reading direction specific) meaning? Some GNOME apps use one over the other; some use both, such as the Terminal:

![](https://linuxhint.com/wp-content/uploads/2019/11/40-7-768x536.png)
